engineio/src/packet.rs

```diff
@@ -1,4 +1,3 @@
-extern crate base64;
```

Comment:
Why is this needed?

Reply:
well……
I have to ask why is this line of code exist in the first place
╮( •́ω•̀ )╭

Reply:
You opened a PR with the title "Fix the implementation for the multi payload". TBH I don't understand why this is necessary for the patch. I like to have clean commits, that do what the title says. If you submit a PR for "applying clippy lints", this would be appropriate. I am still accepting this change, but be aware that in bigger projects, people like clean PRs and no chaos.